src/components/Inputs/TextInput/TextInput.md

```diff
@@ -1,13 +1,16 @@
 ```jsx
-<TextInput
+import validateMinMaxFactory from '../../../validators/validateMinMax'
+import validateTruthy from '../../../validators/validateTruthy'
+;<TextInput
   name="example"
-  minLength={5}
-  maxLength={20}
-  allCaps={true}
   labelCopy="Validation happens after first blur ('touched')—Value's length % 2"
   data-tid="the-text-input"
-  validator={(x) =>
-    x.length % 2 ? 'Text does not have an even number of characters' : ''
-  }
+  validator={(x) => {
```

Comment:
haha, so our 6:00-8:00pm conversation WAS helpful 😋

Reply:
this is pretty much the main case for `validator` turning into `validators` and accepting an array. checking for truthiness is basically step 1 for marking a text input as required, but obviously it wouldn't work on a input which accepts numbers from 0-x. maybe it's a little verbose and requires excessive work from the client (i dont think so) but i sorta like the "composing a list of validator functions" thing and it will probably be appealing to the FP fans out there.